config/kubernetes/sig-docs/teams.yaml

```diff
@@ -352,10 +352,13 @@ teams:
     - abuisine
     - aisonaku
     - annajung
+    - ashnehete # 1.22 RT Docs Shadow
     - awkif
     - bene2k1
     - bradtopol
+    - carlisia # 1.22 RT Docs Shadow
```

Comment:
Hi. The changes look good. How is this list managed (pruned)?

Reply:
@kbhawkey good point, that's on my list to discuss on the next docs meeting. This and `website-milestone-maintainers`.